### PR TITLE
Enable rmi integration tests

### LINKED ISSUE #437: Milestone: docker rmi
Delete images, failing if they are not leaves of the tree or are in an image store that the VCH isn't directly managing.

### PR DESCRIPTION
**Pull request by fdawg4l**
* Removed force test since we don't support untagging of images yet
* Fixed error message to be less horrific when an image is in use

Fixes #437

### CODE REVIEW

**Comment by fdawg4l** [comment]
cc @hickeng and @matthewavery

**Comment by vburenin** [comment]
LGTM

File: tests/test-cases/Group1-Docker-Commands/1-12-Docker-RMI.robot [hunk 1]
```diff
@@ -1,60 +1,45 @@
...
-#    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} images
-#    Should Be Equal As Integers  ${rc}  0
-#    Should Not Contain  ${output}  busybox
-    
-#Remove a fake image
-#    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} rmi fakeImage
-#    Should Be Equal As Integers  ${rc}  1
-#    Should Contain  ${output}  Failed to remove image (fakeImage): Error response from daemon: No such image: fakeImage:latest
\ No newline at end of file
+    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} pull busybox
```
Comment (fdawg4l) [inline tests/test-cases/Group1-Docker-Commands/1-12-Docker-RMI.robot]: > simply not enough.

Not enough for what?  The test is `Basic docker remove image`.  From you comment, I can't tell what aspect of image removal you'd like to exercise.

But yes, if you feel we need to do something _else_, it's likely it should be done in another PR.

Comment (vburenin) [inline tests/test-cases/Group1-Docker-Commands/1-12-Docker-RMI.robot]: just one simple image is not enough to test different code paths including the correctness of removing one image from the set of several.

**Comment by caglar10ur** [comment]
LGTM :sheep: :it:

File: tests/test-cases/Group1-Docker-Commands/1-12-Docker-RMI.robot (hunk 1 above)
Comment (fdawg4l) [inline tests/test-cases/Group1-Docker-Commands/1-12-Docker-RMI.robot]: Oh, that's a good reason. Ill add a test for that shortly.

**Comment by fdawg4l** [comment]
.

**Comment by fdawg4l** [comment]
LGTM

**Pull request merged**